src/js/utils/fullscreenApi.js

```diff
@@ -9,7 +9,7 @@ export const fullscreenApi = {
 };
 
 if (document.fullscreenElement === undefined) {
-  fullscreenApi.enter = "webkitRequestFullScreen";
```

Comment:
I'm running into a bug with this in Safari (Desktop) - it turns out:

* `document.webkitExitFullscreen` is defined, but `document.webkitEnterFullScreen` is not (see screenshot below):

<img width="257" alt="webkit-exit-enter_2021-09-06_01-00-03" src="https://user-images.githubusercontent.com/764988/132181664-810b577b-bc04-4a58-8acc-aa2db3de054a.png">

The way this logic works, in Desktop Safari under these conditions is assigning the `fullscreenApi.enter` to `webkitEnterFullScreen`, which is undefined, leading to the following error:

<img width="1280" alt="enter-not-a-function_2021-09-06_01-01-17" src="https://user-images.githubusercontent.com/764988/132181833-020b9542-8443-4abc-89d1-71ac8d9826b1.png">



This is all in Safari `Version 14.1 (16611.1.21.161.6)`

Reply:
Safari has old and new api's :(
https://github.com/sindresorhus/screenfull/blob/main/index.js#L12-L31